### DISCUSSION TITLE
Unable to integrate Milvus to Spring Boot Application.

### DISCUSSION THREAD

**Discussion by dezynetechnologies** [milvus-io/milvus#34452] [Q&A and General discussion]
I am trying to integrate Milvus with my Spring Boot Application but I am getting status 500: Internal Server error as soon as I create a milvus client. 
This is the code I am writing :
String CLUSTER_ENDPOINT = "http://localhost:19530/";
ConnectConfig connectConfig = ConnectConfig.builder().uri(CLUSTER_ENDPOINT).build();
MilvusClientV2 client = new MilvusClientV2(connectConfig);
Can anyone help me with this error?

**Comment by yhmo** [comment]
Didn't see this error before. 
We got a user report about grcpc-client-spring-boot-autoconfigure 2 years ago: https://github.com/milvus-io/milvus-sdk-java/issues/366, but the error is not like yours.

**Reply by dezynetechnologies** [reply]
"java.lang.NoSuchFieldError: EPOLL_DOMAIN_CLIENT_CHANNEL_TYPE\n\tat io.grpc.netty.shaded.io.grpc.netty.UdsNettyChannelProvider.isAvailable(UdsNettyChannelProvider.java:34)\n\tat io.grpc.ManagedChannelRegistry$ManagedChannelPriorityAccessor.isAvailable(ManagedChannelRegistry.java:211)\n\tat io.grpc.ManagedChannelRegistry$ManagedChannelPriorityAccessor.isAvailable(ManagedChannelRegistry.java:207)\n\tat io.grpc.ServiceProviders.loadAll(ServiceProviders.java:68)\n\tat io.grpc.ManagedChannelRegistry.getDefaultRe…tat org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1791)\n\tat org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)\n\tat org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)\n\tat org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)\n\tat org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\n" 

This is the message I got.
